k9mail/src/main/java/com/fsck/k9/activity/setup/AccountSettings.java

```diff
@@ -353,9 +353,9 @@ public boolean onPreferenceChange(Preference preference, Object newValue) {
 
         mDeletePolicy = (ListPreference) findPreference(PREFERENCE_DELETE_POLICY);
         if (!mIsSeenFlagSupported) {
-            removeListEntry(mDeletePolicy, Integer.toString(Account.DELETE_POLICY_MARK_AS_READ));
+            removeListEntry(mDeletePolicy, Account.DeletePolicy.DELETE_POLICY_MARK_AS_READ.preferenceString());
```

Comment:
Can you please import `DeletePolicy` and all the other enums to avoid overly long expressions.